samples/dashboards/demo/minimal-html/demo.css

```diff
@@ -9,11 +9,19 @@
 
 .row {
     display: flex;
-    flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
This demo doesn't look good:
![image](https://github.com/highcharts/highcharts/assets/57004120/1babab5c-9c00-4966-ae55-af7a8d6b9ebf)